docker/README.md

```diff
@@ -17,7 +19,8 @@ steps to prepare your environment:
   - `POSTGRES_TEST_URL=postgresql://postgres:password123@postgres:5432/usdr_grants_test`
   - You may also have to set the `WEBSITE_DOMAIN` hostname if you are not developing on `localhost`
   ([more info](#cookbook-non-localhost)).
-3. Run `docker compose up -d` to start the services.
+4. Run `docker compose up -d` to start the services in the background (the `-d` flag).
+5. Install application dependencies via yarn: `docker compose exec app yarn`.
```

Comment:
I didn't need to do this (I think [this ](https://github.com/usdigitalresponse/usdr-gost/blob/_staging/docker/Dockerfile#L15)takes care of it?), but if you needed to then +1 for including!

Reply:
Yeah I believe so but in my case I ran tests first — so maybe that’s why. I failed out of tests because of missing deps so I added this.

Reply:
Oh! Okay, I ran into this too but I fixed it by `nohoist: ['mocha']` to the root `package.json`. The problem here was that mocha needs to be in the path, but when yarn workspace invokes commands on that individual workspace, it does so with the workspace's `node_modules`, not the root monorepo. I don't think it affects the deployment one way or another since the app gets packaged separately. 

@TylerHendrickson - mind if I add the nohoist in a PR, or should the workflow be to run the yarn install directly in the workspace?

Reply:
@jakekreider Please feel free to add `nohoist` to a PR – I'm not terribly familiar with that particular directive, but after reading up on it [here](https://classic.yarnpkg.com/blog/2018/02/15/nohoist/) it sounds like a sensible thing to do.

One thing  to mention (disclaimer that I'm not sure how pertinent this is to the change you're proposing) is that we somewhat recently moved away from Lerna in favor of further decoupling `packages/client` and `packages/server` concerns. I'm curious whether you think it would be worth ditching the root-level `package.json` entirely in the spirit of furthering that theme of decoupling?